org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/SealedTypesTests.java

```diff
@@ -4600,7 +4600,7 @@ public void testBug564638b_046() {
 			"3. ERROR in X.java (at line 6)\n" +
 			"	x.<sealed>foo(0);\n" +
 			"	          ^^^\n" +
-			"The parameterized method <sealed>foo(sealed) of type X is not applicable for the arguments (Integer)\n" +
+			"The method foo(sealed) from the type X refers to the missing type sealed\n" +
```

Comment:
What code change causes the type argument from not showing up anymore in the messages ? It wasn't readily obvious

Reply:
Not intentional, but caused by different implementations below `ProblemReporter.invalidatMethod()`:
* `case ParameterizedMethodTypeMismatch` renders signatures with several invocations of `typesAsString()` to fill in "`<{3}>{0}({1})`"
* `case MissingTypeInSignature` (new) delegates to the pre-existing method `missingTypeInMethod()` which is agnostic to type parameters

We could make `missingTypeInMethod()` typeparameter-aware, which could require to update another set of test expectations. Is it worth it?

Reply:
> Not intentional, but caused by different implementations below `ProblemReporter.invalidatMethod()`:
> 
> * `case ParameterizedMethodTypeMismatch` renders signatures with several invocations of `typesAsString()` to fill in "`<{3}>{0}({1})`"
> * `case MissingTypeInSignature` (new) delegates to the pre-existing method `missingTypeInMethod()` which is agnostic to type parameters
> 
> We could make `missingTypeInMethod()` typeparameter-aware, which could require to update another set of test expectations. Is it worth it?

No, not necessary.